README.md

```diff
@@ -1,22 +1,7 @@
-## Mento SDK
+# Mento SDK
 
-This repo contains the source code of the software development kit for the Mento protocol. The repository is built with npm which is used for the compilation, testing and deployment of the sdk.
+The official Mento Protocol SDK for interacting with Multi-Collateral Mento smart contracts on the Celo network.
 
-## Getting Started
+# In depth docs
 
-```bash
-# Get the latest code
-git clone git@github.com:mento-protocol/mento-sdk.git
-
-# Change directory to the the newly cloned repo
-cd mento-sdk
-
-# Install dev dependencies with yarn
-yarn
-
-# Build the sdk with yarn
-yarn build
-
-# Run all tests with yarn
-yarn test
-```
+- [ ] TODO: Adds docs link
```

Comment:
Note, the contents of this Readme will be what's shown on NPM when you publish, so info oriented more to SDK users may be a good idea. Feel free to ignore if this is temporary.

Reply:
Yeah let's put in at least a bit of work on the readme to make it a bit more friendly to SDK consumers.

Reply:
Chatted with Bogdan about this, we'll create a separate issue to write the docs and add them to https://docs.mento.org/ and will link it here afterwards